### DISCUSSION TITLE
How can I enable language which is not written in `app/config/firefly.php` file

### DISCUSSION THREAD

**Discussion by muuky** [firefly-iii/firefly-iii#7545] [Firefly III] [answered]
Finally found a web application that's perfect for me.

I tried to use an application that was perfect for me, and also found that a Korean translation was 100% on [cardwin](https://crowdin.com/project/firefly-iii).

However, when I launched firefly, Korean was not present in the language selection.

So, not only checked if the Korean translation was included in the docker container (`/var/www/html/resources/lang`) but also checked how to enable it through the [documentation](https://docs.firefly-iii.org/firefly-iii/faq/translate/).

I found that the actual `/var/www/html/app/config/firefly.php` file was not in the location written in the document, and that the location was `/var/www/html/config/firefly.php`.

Anyway, in the php file, I couldn't find Korean among the commented parts described in the document.

**In conclusion, I wonder if there is a way to activate the language other than un-commenting to activate it.**

**Comment by JC5** [comment, marked as answer]
I wasn't aware that it was already at 100%. I shall enable it in the next release.

**Reply by muuky** [reply]
thanks for fast replay, it was 100% yesterday... I just found it became 99%. I will fill 1%
